### PR TITLE
FIX: Typos in Comms social media role-handbook

### PR DESCRIPTION
**Pull request by SD-13**
Only fix some typos.

### CODE REVIEW

File: communication/contributor-comms/role-handbooks/Social-Media.md [hunk 1]
```diff
@@ -13,28 +13,28 @@ Do note that tweets targeting the greater Kubernetes community should potentiall
...
 
 Team leads: 3-5 hours per week.
-Team members: 1-3 hour per week.
+Team members: 1-3 hours per week.
 
 ## General Expectations
 
 - Rules for social posts and more can be found in [social guidelines](https://github.com/kubernetes/community/blob/master/communication/contributor-comms/storytelling-resources/social-guidelines.md)
 - Be welcoming, be yourself
-- Understanding, communicating, and setting/adjusting [social guidelines](https://github.com/kubernetes/community/blob/master/communication/contributor-comms/storytelling-resources/social-guidelines.md)
```
Comment (cblecker) [inline communication/contributor-comms/role-handbooks/Social-Media.md:29]: This is not a typo. For more information: https://en.wikipedia.org/wiki/Serial_comma

File: communication/contributor-comms/role-handbooks/Social-Media.md [hunk 2]
```diff
@@ -13,28 +13,28 @@ Do note that tweets targeting the greater Kubernetes community should potentiall
...
+- Understanding, communicating and setting/adjusting [social guidelines](https://github.com/kubernetes/community/blob/master/communication/contributor-comms/storytelling-resources/social-guidelines.md)
 - Coordinating work to create tweets, ensuring tweet requests are fulfilled
 - Working with other Contributor Comms leads, such as the Event Coordinator Lead, on social media tasks
 - Approving tweet PRs on [kubernetes-sigs/contributor-tweets](https://github.com/kubernetes-sigs/contributor-tweets)
 - Training new leads
 
-The leader(s) of the social media team will be expected to monitor the @K8sContributors handle. Retweets, replies, and responding to replies appropriately fall under the purview of the social media lead. Time-sensitive tweets should also be handled directly by the team lead. Team leads should coordinate with team members to plan tweets in advance, particularly for large contributor-focused events such as KubeCon or Contributor Summit.
```
Comment (cblecker) [inline communication/contributor-comms/role-handbooks/Social-Media.md:35]: This is not a typo. For more information: https://en.wikipedia.org/wiki/Serial_comma

File: communication/contributor-comms/role-handbooks/Social-Media.md [hunk 3]
```diff
@@ -43,4 +43,4 @@ Leads should work with shadows to understand their capacity to take on the needs
 * Becoming an owner (approver privileges) on the [kubernetes-sigs/contributor-tweets](https://github.com/kubernetes-sigs/contributor-tweets) repo
 * Watching for new tweet PRs and working with PR creators on scheduling/approving tweets
 * Watching for new tweet requests and ensuring they get assigned to team members or created directly
-* Taking on direct Twitter management
+* Taking on a direct Twitter management
```
Comment (cblecker) [inline communication/contributor-comms/role-handbooks/Social-Media.md:46]:
```suggestion
* Taking on direct management of a social media account
```

**Review by cblecker — CHANGES_REQUESTED** [review]

File: communication/contributor-comms/role-handbooks/Social-Media.md (hunk 3 above)
Comment (SD-13) [reply communication/contributor-comms/role-handbooks/Social-Media.md]: done

File: communication/contributor-comms/role-handbooks/Social-Media.md (hunk 2 above)
Comment (SD-13) [reply communication/contributor-comms/role-handbooks/Social-Media.md:35]: Thanks for the info, I have reverted this change but not sure why it is not showing `outdated`.

File: communication/contributor-comms/role-handbooks/Social-Media.md (hunk 1 above)
Comment (SD-13) [reply communication/contributor-comms/role-handbooks/Social-Media.md]: I have reverted this change too

**Comment by SD-13** [comment]
Hi @cblecker, I have addressed all your comments. PTAL, Thanks,

**Comment by cblecker** [comment]
/lgtm
/approve

Thanks! 🚀

**Pull request merged**